docs/CdC.md

```diff
@@ -0,0 +1,28 @@
+# Cahier des charges
```

Comment:
This file could be removed.
The different tasks/stories will be on the project dashboard

Reply:
Done